### ISSUE TITLE
Fuzzy icons on scaled MacBook Pro Retina Display

### ISSUE THREAD

**Issue by connorwil** [rsms/fb-mac-messenger#366]
Icons appear pixelated when you have a higher 'looks like' display resolution on MacBook Pro 15"

**Comment by dangh** [comment]
@connorwil can you help me to test my build [here](https://github.com/dangh/fb-mac-messenger/releases/tag/v0.1.12.2017-03-15_01-56-40)? If it's work well then I will create a pull request.

**Comment by themightykitten** [comment]
Yep same here. I can make new ones if rsms is willing to implement those
